node/src/main/resources/files/nms/server.properties

```diff
@@ -18,40 +18,38 @@ difficulty=easy
 network-compression-threshold=256
 max-tick-time=60000
 require-resource-pack=false
-max-players=20
 use-native-transport=true
-enable-status=true
+max-players=20
 online-mode=false
+enable-status=true
 allow-flight=false
 broadcast-rcon-to-ops=true
 view-distance=10
-max-build-height=256
-resource-pack-prompt=
 server-ip=
-allow-nether=false
+resource-pack-prompt=
+allow-nether=true
 server-port=25565
-sync-chunk-writes=true
 enable-rcon=false
+sync-chunk-writes=true
 op-permission-level=4
 prevent-proxy-connections=false
 hide-online-players=false
 resource-pack=
 entity-broadcast-range-percentage=100
 simulation-distance=10
-player-idle-timeout=0
 rcon.password=
-force-gamemode=false
+player-idle-timeout=0
 debug=false
+force-gamemode=false
 rate-limit=0
 hardcore=false
 white-list=false
 broadcast-console-to-ops=true
-previews-chat=true
 spawn-npcs=true
+previews-chat=false
 spawn-animals=true
-snooper-enabled=true
 function-permission-level=2
-level-type=default
+level-type=minecraft\:normal
```

Comment:
does this work in older versions like 1.8.8 too?

Reply:
sure